TYPO3.Flow/Classes/TYPO3/Flow/Error/Debugger.php

```diff
@@ -175,7 +175,7 @@ protected static function renderObjectDump($object, $level, $renderProperties =
         $scope = '';
         $additionalAttributes = '';
 
-        if ($object instanceof \Doctrine\Common\Collections\Collection) {
+        if ($object instanceof \Doctrine\Common\Collections\Collection || $object instanceof \ArrayObject) {
```

Comment:
This might make sense, but it's not related to this feature so I'd suggest to skip it or extract it to its own PR